### DISCUSSION TITLE
Αδυναμία Παρακολούθησης ΟΣΣ 22 Δεκ

### DISCUSSION THREAD

**Discussion by VagNta** [sdy60/syllabus#178] [Q&A] [answered]
Καλημέρα, σας ενημερώνω πως θεωρώ αδύνατο αυτή την στιγμή να παρακολουθήσω την ΟΣΣ της 22 Δεκ. γιατί είμαι άρρωστος, αν είμαι κάπως καλύτερα αύριο, θα το προσπαθήσω. 
Καλή Συνέχεια.

**Comment by epidrome** [comment, marked as answer]
Θα γίνουν τουλάχιστον δύο ακόμη ΟΣΣ3
